features/facebookImport/src/views/explore/explore.jsx

```diff
@@ -10,20 +10,23 @@ const AnalysisCard = ({
     exploreScrollingProgress,
 }) => {
     return (
-        <div className="analysis-card">
-            <h1>{analysis.title}</h1>
-            <div>{analysis.renderSummary()}</div>
-            {analysis.renderDetails ? (
-                <RouteButton
-                    route="/explore/details"
-                    className="details-button"
-                    onClick={() => setActiveDetails(analysis)}
-                    stateChange={{ exploreScrollingProgress }}
-                >
-                    View details
-                </RouteButton>
-            ) : null}
-        </div>
+        <>
+            <div className="analysis-card">
+                <h1>{analysis.title}</h1>
+                <div className="summary-text">{analysis.renderSummary()}</div>
+                {analysis.renderDetails ? (
+                    <RouteButton
+                        route="/explore/details"
+                        className="details-button"
+                        onClick={() => setActiveDetails(analysis)}
+                        stateChange={{ exploreScrollingProgress }}
+                    >
+                        View details
+                    </RouteButton>
+                ) : null}
+            </div>
+            <div className="card-separator"></div>
+        </>
```

Comment:
... together with this?

Reply:
I wrapped the content using the React fragment (<> ... </>) in order to include the "card-separator" line and to avoid adding too many `div` that could be very annoying to center (if that's what you mean with your question).

If you think that we should delete the React fragments, we could also the "card-separator" and add a `border-top` in CSS, depending on how the final design should look like.

Reply:
So that's a React fragment :-) I had no idea. Thanks for the explanation, and sorry to bother you with that.

Reply:
No problem 😄